### PR TITLE
added pack switch minimum-from-lock-file 

### PR DESCRIPTION
**Pull request by pms1969**
which adds the minimum version for a packages dependencies from the built against version of the package.

includes integration test.

### CODE REVIEW

**Comment by forki** [comment]
did you take a look at the build results?

**Comment by pms1969** [comment]
No I hadn't.  Was a late check in last night, and been out this morning. I'll take a look in a few hours.

**Comment by pms1969** [comment]
Looks like the binaries didn't get added.  As soon as I get back to my laptop, I'll commit them..

**Comment by pms1969** [comment]
Is this ok?

**Comment by forki** [comment]
will take a look tomorrow.

2016-02-02 21:12 GMT+01:00 Paul Saunders <notifications@github.com>:

File: src/Paket.Core/PackageMetaData.fs [hunk 1]
```diff
@@ -265,13 +265,33 @@ let findDependencies (dependencies : DependenciesFile) config platform (template
                 match group with
                 | None ->
                     match version with
-                    | Some v -> np.Name,VersionRequirement.Parse (v.ToString())
-                    | None -> np.Name,VersionRequirement.AllReleases
+                    | Some v -> 
+                        tracefn "HAA => %s | %s" (np.Name.ToString()) (v.ToString())
+                        np.Name,VersionRequirement.Parse (v.ToString())
+                    | None -> 
+                        trace "It really was null"
```
Comment (forki) [inline src/Paket.Core/PackageMetaData.fs]: tu

File: src/Paket.Core/PackageMetaData.fs [hunk 2]
```diff
@@ -265,13 +265,33 @@ let findDependencies (dependencies : DependenciesFile) config platform (template
...
+                        let lockedVersion() = 
+                            let findLockedGroup() =
+                                lockFile.GetDependencyLookupTable()
+                                |> Seq.map (fun m -> m.Key)
```
Comment (forki) [inline src/Paket.Core/PackageMetaData.fs]: this looks way to complex.

File: src/Paket.Core/PackageMetaData.fs (hunk 1 above)
Comment (forki) [inline src/Paket.Core/PackageMetaData.fs]: ok let me merge and clean it.

**Comment by forki** [comment]
Please review https://github.com/fsprojects/Paket/pull/1452

**Comment by pms1969** [comment]
bit snowed at work at the moment, will pull it down and have a look as soon as I'm free later.

**Pull request closed without merging**